ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,44 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+
+{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+#### TCP MSS Ceiling
+
+| Protocol | Segment Size |
+| -------- | -------------|
+{%      if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}| ipv4 | {{ system.control_plane.tcp_mss.ipv4 }} |
+{%      endif %}
+{%      if system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}| ipv6 | {{ system.control_plane.tcp_mss.ipv6 }} |
+{%      endif %}
+{%    endif %}
+
+{%    if system.control_plane.ipv4_access_group is arista.avd.defined or system.control_plane.ipv6_access_group is arista.avd.defined %}
+#### Control-Plane Access-Groups
+
+| Protocol | VRF | Access-list |
+| -------- | --- | ------------|
+{#      IPv4 Access-groups #}
+{%      if system.control_plane.ipv4_access_group is arista.avd.defined %}
+{%        for acl_set in system.control_plane.ipv4_access_group | arista.avd.natural_sort %}
+{%          if acl_set.vrf is arista.avd.defined and acl_set.vrf is not none %}| IPv4 | {{ acl_set.vrf }} | {{ acl_set.acl_name }} |
```

Comment:
move table content to it's own line. Same in other tables.
`is not none` is already part of `arista.avd.defined` so can be removed.

Reply:
Done